pkg/reckoner/plot.go

```diff
@@ -20,16 +20,15 @@ import (
 	"os"
 	"strings"
 
-	"gopkg.in/yaml.v3"
-	"k8s.io/klog/v2"
+	"gopkg.in/yaml.v2"
 
 	"github.com/fairwindsops/reckoner/pkg/course"
 	"github.com/fatih/color"
 	"github.com/thoas/go-funk"
 )
 
 // Plot actually plots the releases
-func (c Client) Plot() error {
+func (c *Client) Plot() error {
```

Comment:
Confirm why this was changed.

Reply:
I changed this because `c.Continue()` uses a pointer to a client object. It doesn't seem to be a syntax error if we change it back to a non-pointer but I'm not really sure the repercussions.

Reply:
Technically, it only needs to be a pointer if you're going to modify it inside the function. I've seen a general convention to just make a it a pointer though, so it's probably fine?